content/python/concepts/built-in-functions/terms/abs/abs.md

```diff
@@ -10,30 +10,37 @@ Tags:
   - 'Strings'
 CatalogContent:
   - 'learn-python-3'
-  - 'paths/computer-science'
   - 'paths/data-science'
 ---
 
-Takes in a numeric data type such as `int` or `float` and returns the absolute value of the argument.
+The `abs()` function returns the absolute value of a numeric argument.
 
 ## Syntax
 
-```py
+```pseudo
 abs(n)
 ```
 
-## Example 1
+Where n is a real number, for example an int or float, that will have its absolute value taken. The absolute value of n will be the non-negative value of n, a positive value will remain unchanged and a negative value will have its negative sign removed.
```

Comment:
```suggestion
The return value will be the absolute value of the `n` parameter, which is of type `int` or `float`.

The absolute value of `n` will be its distance from zero regardless of its direction (i.e., whether it is positive or negative). Since the absolute value is never negative, a positive value will remain unchanged and a negative value will have its negative sign removed
```